selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaIde.java

```diff
@@ -181,6 +182,9 @@ public void switchToIdeFrame() {
   public void pressKeyCombination(CharSequence... combination) {
     final String keyCombination = chord(asList(combination));
 
+    // for ensure that events are end before pressing keys combination
```

Comment:
slight correction `// to ensure that events are ended before pressing keys combination`